cmake/Tools/Platform/Android/android_support.py

```diff
@@ -854,6 +863,9 @@ def create_lumberyard_app(self, project_dependencies):
             if self.override_ninja_path:
                 cmake_argument_list.append(f'"-DCMAKE_MAKE_PROGRAM={common.normalize_path_for_settings(self.override_ninja_path)}"')
 
+            if self.oculus_project:
+                cmake_argument_list.append('"-DANDROID_USE_OCULUS_OPENXR=ON"')
```

Comment:
Where is this Cache variable declared at?
I don't see it as part of the Oculus OpenXR [documentation](https://developer.oculus.com/documentation/native/android/mobile-build-run-hello-xr-app/)

Reply:
It's in the other PR: https://github.com/o3de/o3de-extras/pull/14